libs/base/utilities/src/decorators/hydratable/hydratable.ts

```diff
@@ -111,14 +124,19 @@ function hydratableClass<T extends Type<HTMLElement>>(
     render(): TemplateResult {
       const states = this[asyncStates];
 
+      setTimeout(() => {
+        if (!this.useRealRender) this.useRealRender = true;
```

Comment:
I'm still wondering, if this one is fully bullet proof, as we are not triggering next render explicitely?